libs/cms/src/lib/environments/environment.ts

```diff
@@ -53,6 +54,7 @@ export default {
     'menuLink',
     'menuLinkWithChildren',
     'menu',
+    'grant',
```

Comment:
Is a grant really a "nested" content type? Is there something that needs to be re-indexed into elastic that's "above" the grant (stores a list of grants for example) in Contentful?

Reply:
No, this doesn't need to be there